kong/init.lua

```diff
@@ -281,6 +285,8 @@ local function execute_plugins_iterator(plugins_iterator, phase, ctx)
     if old_ws then
       ctx.workspace = old_ws
     end
+
+    ::continue::
```

Comment:
this is a behaviour change: these two things (`reset_log` and resetting the workspace) are currently executed in the `delay_response and ctx.delayed_response` case, but with this patch they're skipped.  is that a desired change?